OutOfSchool/OutOfSchool.AuthServer.Tests/Extensions/EmailSenderExtensionsTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using OutOfSchool.AuthorizationServer.Config;
+using OutOfSchool.AuthorizationServer.Extensions;
+using OutOfSchool.EmailSender.Quartz;
+using OutOfSchool.EmailSender.Services;
+using Quartz;
+
+namespace OutOfSchool.AuthServer.Tests.Extensions;
+
+[TestFixture]
+public class EmailSenderExtensionsTests
+{
+    [Test]
+    public async Task AddEmailSender_WithValidConfig_AddsJobAndTrigger()
```

Comment:
(Test naming) Name part `_AddsJobAndTrigger` promise to check adding job and trigger, but in the code only job existence is asserted. Please either change the name or add trigger existence check.